### PR TITLE
METRON-2348 fix prepare-commit

### PR DESCRIPTION
**Pull request by ottobackwards**
Fix the prepare-commit script by changing the check for "Status" to
instead two checks, for the PR being open, and it being cleanly
mergable.
NOTE: this adds a dependency on [jq](https://stedolan.github.io/jq/)

Testing : run prepare-commit for a PR that exists in either repo, see that it doesn't work.
run again with this fix

> NOTE:  this pr introduces a requirement for [jq](https://stedolan.github.io/jq/)

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [x] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [-] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```

- [-] Have you written or updated unit tests and or integration tests to verify your changes?
- [-] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [-] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### For documentation related changes:
- [-] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

- [-] Have you ensured that any documentation diagrams have been updated, along with their source files, using [draw.io](https://www.draw.io/)? See [Metron Development Guidelines](https://cwiki.apache.org/confluence/display/METRON/Development+Guidelines) for instructions.

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

---
This change is [Reviewable](https://reviewable.io/reviews/apache/metron/1580)

### CODE REVIEW

**Comment by JonZeolla** [comment]
+1 by inspection, pending successful CI

**Comment by JonZeolla** [comment]
FYI tested this to merge apache/metron-bro-plugin-kafka#42 successfully.

**Comment by ottobackwards** [comment]
@JonZeolla can you check the latest.  If we can't commit when unstable, we can't fix anything.

File: dev-utilities/committer-utils/metron-committer-common [hunk 1]
```diff
@@ -135,8 +135,8 @@ function read_pull_request {
       exit 1
     fi
 
-    if [[ "$PR_MERGEABLE" != "\"clean\"" ]]; then
-      echo "Error: pull request #$PR is not mergable"
+    if [[ "$PR_MERGEABLE" == "\"clean\"" ]] || [[ "$PR_MERGEABLE" == "unstable" ]]; then
```
Comment (JonZeolla) [inline dev-utilities/committer-utils/metron-committer-common:138]: I think you meant `if [[ "$PR_MERGEABLE" != "\"clean\"" ]] || [[ "$PR_MERGEABLE" == "unstable" ]]; then`

Comment (ottobackwards) [reply dev-utilities/committer-utils/metron-committer-common:138]: drat

**Comment by JonZeolla** [comment]
+1

**Comment by ottobackwards** [comment]
@JonZeolla one last check please

**Comment by JonZeolla** [comment]
+1 stands

**Pull request closed without merging**